guides/Pagination.md

```diff
@@ -18,37 +18,71 @@ a new field and order.
 
 ### Sort _List_ endpoints
 
-_List_ endpoints are designed to be lightweight HTTP GET requests.
+_List_ endpoints are designed to be lightweight requests.
 For this reason, sorting is applied through the query parameters,
-typically with `sort.fieldName` and `sort.order`.
+typically with sort field name and order fields.
 
-For example, to list contacts by last name in ascending order,
-append these query parameters to the request:
+For example, to list contacts by last name in ascending order:
 
+::::tabs
+:::REST_TAB
 ```txt
 ?sort.fieldName=info.name.last&sort.order=ASC
 ```
+:::
+:::SDK_TAB
```

Comment:
full code sample:
import { listContacts } from 'wix-crm-backend';

const sort = [
  { fieldName: "info.name.last", order: "ASC" },
];

listContacts({ sort })
  .then(results => {
    // results.items is sorted as requested
    console.log(results.items);
  })
  .catch(error => {
    console.error(error);
  });